source/content/debug-cache.md

```diff
@@ -48,53 +48,52 @@ Generated by WordPress or Drupal to send data back to the browser. The platform
 </dl>
 
 ## Debug Caching Issues
-
 Understand caching behavior for a given page by analyzing cache related HTTP headers from the command line with `curl`. For example:
 
-```bash{outputLines: 2-20}
+```bash{outputLines: 2-18}
 curl -I https://www.example.com
 HTTP/2 200
-cache-control: public, max-age=600 // highlight-line
+cache-control: public, max-age=0 // highlight-line
 content-type: text/html; charset=UTF-8
 server: nginx
-//highlight-start
-set-cookie: cookielawinfo-checkbox-necessary=yes; expires=Thu, 20-Feb-2020 17:31:51 GMT; Max-Age=3600; path=/
-set-cookie: cookielawinfo-checkbox-non-necessary=yes; expires=Thu, 20-Feb-2020 17:31:51 GMT; Max-Age=3600; path=/
-//highlight-end
 x-pantheon-styx-hostname: styx-fe1-a-789d66bff9-tztp6
 x-styx-req-id: 7f93c166-53fe-11ea-803e-b26d7703e33f
-date: Thu, 20 Feb 2020 16:31:51 GMT
-x-served-by: cache-mdw17379-MDW, cache-chi21146-CHI
-x-cache: MISS, MISS
-x-cache-hits: 0, 0
-x-timer: S1582216311.492451,VS0,VE204
-vary: Accept-Encoding, Cookie, Cookie
+date: Tue, 08 Dec 2020 16:41:37 GMT
+x-served-by: cache-mdw17348-MDW
+x-cache: MISS
+x-cache-hits: 0
+x-timer: S1607445693.448541,VS0,VE4134
+vary: Accept-Encoding, Cookie
+x-robots-tag: noindex
 age: 0 // highlight-line
 accept-ranges: bytes
 via: 1.1 varnish
+content-length: 41278
 ```
 
-If you see `Age: 0` after multiple requests, your site is not caching properly.
+This specific example is from a Drupal site that has anonymous caching enabled, but has "none" set for expiration of cached pages within Drupal's performance settings (`/admin/config/development/performance`). For details, see [Drupal Performance and Caching Settings](/drupal-cache).
 
-This particular example is covered in [WordPress Plugins and Themes with Known Issues](/plugins-known-issues#gdpr-cookie-consent).
+### Set-Cookie Headers
+Pantheon's platform will not cache a response that contains the `set-cookie` header.
 
-### Cookie Name Prefix
+<Alert title="Note" type="info" >
 
-Pantheon's platform will not cache a response that contains the `set-cookie` header.
+Do not use Curl to debug this scenario. Instead, investigate `set-cookie` issues and fixes from your browser, for instructions see [Test Global CDN with Browser Headers](/test-global-cdn-caching#test-global-cdn-with-browser-headers). Curl does not handle cookie data in the way that your browser does and effectively misrepresents cookie and caching behavior as a result.
 
-The best way to utilize cookies on Pantheon is by having the cookie name match the `STYXKEY[a-zA-Z0-9_-]+` naming convention, and loading them in the first load, not on every page load. Refer to the sample code outlined in [Working with Cookies on Pantheon](/cookies#cache-varying-cookies).
+</Alert>
 
-For example, if a theme file sets a cookie as:
+Search for instances across the site's codebase and once isolated, edit the cookie name so that is matches the [`STYXKEY[a-zA-Z0-9_-]+` naming convention](/caching-advanced-topics#using-styxkey).
```

Comment:
instances of what, precisely?

Reply:
Of the cookie name they're seeing in their browser, in the set-cookie header of the response